README.md

```diff
@@ -134,10 +134,49 @@ services:
     - ...
 
 ```
+### Start with a distro backup
+
+To restore the backup unzip the the content and proceed with the following steps:
+- To restore OpenMRS database check [here](#start-with-a-custom-mysql-dump)
```

Comment:
- Restore OpenMRS backup following the steps described here.
- Restore Odoo and OpenELIS backup following the steps described here.
- Finally, restore the filestore (Odoo filestore, OpenMRS config checksums...) following steps as described here.